.packit.yaml

```
@@ -0,0 +1,29 @@
+specfile_path: ansible-lint.spec
+actions:
+  create-archive:
+    - "python3 setup.py sdist --dist-dir ."
+    - "sh -c 'echo ansible-lint-$(python3 setup.py --version).tar.gz'"
```

@TomasTomecek would something like this work?
```suggestion
    - "sh -c 'echo ${PACKIT_PROJECT_NAME_VERSION}.tar.gz'"
```
Is this var populated from `get-current-version`? (The docs imply that it's only coming from `git-describe` which I want to clarify)

---

yup, you're correct, the content of the env var is by default coming from `git describe` but you can override with the `get-current-version` action